base/abstractarraymath.jl

```diff
@@ -220,6 +220,9 @@ reim(A::AbstractArray)
 +(x::AbstractArray{<:Number}) = x
 *(x::AbstractArray{<:Number,2}) = x
 
+rad2deg(A::AbstractArray) = broadcast_preserving_zero_d(rad2deg, A)
+deg2rad(A::AbstractArray) = broadcast_preserving_zero_d(deg2rad, A)
```

Comment:
I thought a long-term goal was to have fewer implicitly-broadcast functions, not more